Resources/translations/SonataAdminBundle.fr.xliff

```diff
@@ -174,6 +174,10 @@
                 <source>flash_batch_delete_success</source>
                 <target>Les éléments séléctionnés ont été supprimés avec succès.</target>
             </trans-unit>
+            <trans-unit id="flash_batch_delete_no_element">
+                <source>flash_batch_delete_no_element</source>
+                <target>Aucun élément supprimés.</target>
```

Comment:
> Aucun élements supprimés.

or 

> Aucun élément supprimé.

Reply:
The second one please.

Reply:
Oups, `Aucun élément supprimé.` should do the trick